views/index.pug

```diff
@@ -0,0 +1,19 @@
+html 
+    body 
+      h1 KIM'S CONVENIENCE
+  h2 Developed By
+      div
```

Comment:
You don't need these divs, h2 and ul are block elements. If you need more room between them you use margin for that.